src/query/catalog/src/plan/pushdown.rs

```diff
@@ -63,6 +63,8 @@ pub struct PrewhereInfo {
     /// filter for prewhere
     /// Assumption: expression's data type must be `DataType::Boolean`.
     pub filter: RemoteExpr<String>,
+    /// inverted_filter  for prewhere
+    pub inverted_filter: RemoteExpr<String>,
```

Comment:
`inverted_filter` is not always needed. How about change this to an `Option`.

For example, the `do_mutation_block_pruning` call in `FuseTable::try_add_update_source` doesn't need `inverted_filter`.